### PR TITLE
Match More functions

### PR DESCRIPTION
**Pull request by PokeCodec**
I merged the rest of my dozens of branches into one.

Better this than 6 or more PRs

### CODE REVIEW

**Comment by GriffinRichards** [comment]
Doesn't match. Please test these locally

**Comment by PokeCodec** [comment]
Alright, fixed!

File: include/constants/rgb.h [hunk 1]
```diff
@@ -1,6 +1,10 @@
 #ifndef GUARD_RGB_H
 #define GUARD_RGB_H
 
+#define R(color) ((color) & 0x1F)
+#define G(color) (((color) >> 5) & 0x1F)
+#define B(color) (((color) >> 10) & 0x1F)
```
Comment (GriffinRichards) [inline include/constants/rgb.h:6]: These names imply to-RGB, but they're from-RGB. i.e. I'd expect `R(x) | G(y) | B(z)` to be equivalent to `RGB(x, y, z)`

File: src/frontier_util.c [hunk 2]
```diff
@@ -2509,11 +2508,13 @@ void CreateFrontierBrainPokemon(void)
...
             continue;
-
         do
         {
-            j = Random32();
-        } while (IsShinyOtIdPersonality(FRONTIER_BRAIN_OTID, j) || sFrontierBrainsMons[facility][symbol][i].nature != GetNatureFromPersonality(j));
+            do
+            {
+                j = Random32(); //Should be one while loop, but that doesn't match
+            } while (IsShinyOtIdPersonality(FRONTIER_BRAIN_OTID, j)); //See above comment
```
Comment (GriffinRichards) [inline src/frontier_util.c:2516]: Second comment is unnecessary

File: src/frontier_util.c [hunk 3]
```diff
@@ -2535,243 +2536,7 @@ void CreateFrontierBrainPokemon(void)
...
-    mov r9, r4\n\
-    mov r10, r5\n\
-    pop {r4-r7}\n\
-    pop {r0}\n\
-    bx r0\n\
-    .pool\n\
-");
-}
-#endif
+
```
Comment (GriffinRichards) [inline src/frontier_util.c:2539]: Extra newline

File: src/pokedex.c [hunk 4]
```diff
@@ -5236,84 +5236,24 @@ static void Task_ExitSearchWaitForFade(u8 taskId)
...
-        temp[x + i + 32] |= flags * 0x1000;
+        temp = *(u16 *)(ptr + (y+0)*64 + (x+i)*2);
+		temp &= 0x0fff;
+		temp |= (flags << 12);
+		*(u16 *)(ptr + (y+0)*64 + (x+i)*2) = temp;
+   		
+		temp = *(u16 *)(ptr + (y+1)*64 + (x+i)*2);
+		temp &= 0x0fff;
+		temp |= (flags << 12);
+		*(u16 *)(ptr + (y+1)*64 + (x+i)*2) = temp;
```
Comment (GriffinRichards) [inline src/pokedex.c:5254]: Tabs

File: src/pokenav_main_menu.c [hunk 5]
```diff
@@ -471,140 +469,42 @@ void sub_81C7990(u32 a0, u16 a1)
...
+    {
+        CpuCopy16(a1, palette, a2 * 2);
+    }
+    else
+    {
+        int r, g, b;
+        int r1, g1, b1;
+        while (a2--)
+        {
+
```
Comment (GriffinRichards) [inline src/pokenav_main_menu.c:489]: More unnecessary whitespace

File: src/pokenav_main_menu.c [hunk 6]
```diff
@@ -471,140 +469,42 @@ void sub_81C7990(u32 a0, u16 a1)
...
-    mov r8, r3\n\
-    mov r9, r4\n\
-    mov r10, r5\n\
-    pop {r4-r7}\n\
-    pop {r0}\n\
-    bx r0\n\
-    .syntax divided");
+void sub_81C79BC(const u16 *a0, const u16 *a1, int a2, int a3, int a4, u16 *palette)
+{
+
```
Comment (GriffinRichards) [inline src/pokenav_main_menu.c:474]: Unnecessary whitespace

**Review by GriffinRichards — CHANGES_REQUESTED** [review]

**Comment by PokeCodec** [comment]
Fixed!

**Comment by PokeCodec** [comment]
I am 100% ready for this now!

**Pull request closed without merging**